src/wrappers/express/ExpressExecutor.ts

```diff
@@ -65,15 +66,18 @@ export function finishTrace(pluginContext: PluginContext, execContext: Execution
 }
 
 function handleRoutePath(context: ExecutionContext, resourceName: string) {
-    const { rootSpan, request, response } = context;
+    const { rootSpan, response } = context;
 
-    const triggerOperationName = request.hostname + resourceName;
+    const triggerOperationName = resourceName;
     context.triggerOperationName = triggerOperationName;
 
     // Change root span name and response header
     rootSpan.operationName = resourceName;
     response.set(TriggerHeaderTags.RESOURCE_NAME, triggerOperationName);
     InvocationSupport.setAgentTag(SpanTags.TRIGGER_OPERATION_NAMES, [triggerOperationName]);
+    ApplicationManager.getApplicationInfoProvider().update({
```

Comment:
I think, `applicationResourceName` should be stored into `ExecutionContext` and invocation data should be feed from there